Libplanet/Tx/InvalidTxException.cs

```diff
@@ -16,12 +16,32 @@ public abstract class InvalidTxException : Exception
         /// the <see cref="Exception.Message"/> string.</param>
         /// <param name="message">A descriptive error message for programmers.
         /// </param>
-        protected InvalidTxException(TxId txid, string message)
+        protected InvalidTxException(
+            TxId txid,
+            string message)
             : base($"{txid.ToHex()}: {message}")
         {
             TxId = txid;
         }
 
+        /// <summary>
+        /// Instantiates a new exception object with proper metadata.
+        /// </summary>
+        /// <param name="txid">The invalid <see cref="Transaction{T}"/>'s
+        /// <see cref="Transaction{T}.Id"/>.  It is automatically included to
+        /// the <see cref="Exception.Message"/> string.</param>
+        /// <param name="message">A descriptive error message for programmers.
+        /// </param> <param name="innerException">The <see cref="Exception"/> for
+        /// <see cref="Exception.InnerException"/>.</param>
```

Comment:
trivial, but...
```suggestion
        /// </param> 
        /// <param name="innerException">The <see cref="Exception"/> for
        /// <see cref="Exception.InnerException"/>.</param>
```

Reply:
And like, this?

https://github.com/colibrishin/libplanet/blob/9c8fe16a2bb18b8e239130c2285c98e52e79af59/Libplanet/Tx/TxPolicyViolationException.cs#L32-L34